transformations/transformations/utility_routines.py

```diff
@@ -100,6 +100,9 @@ def transform_subroutine(self, routine, **kwargs):
         """
         Apply transformation to subroutine object
         """
+        if 'item' in kwargs and kwargs['item'].local_name != routine.name.lower():
```

Comment:
[no action] This is no longer needed once PR #100 lands. No need to take out here, just "note to self" to mop this up in the rebase!